packages/react/src/components/AccountSettings/DeleteUser/types.ts

```diff
@@ -11,6 +12,8 @@ export interface WarningViewProps {
   onConfirm: () => void;
   /** whether account deletion is in progress */
   isDisabled: boolean;
+  /** overrides default display text */
+  displayText?: Partial<DeleteUserDisplayText>;
```

Comment:
To improve readability in intellisense and our docs, what do you think of creating `DeleteUserDisplayText` with optional keys and extending that with `Required` for usage in creation of the default text? For example:
```ts
interface DeleteUserDisplayText {
  cancelText?: string;
  deleteAccountText?: string; 
  warningText?: string;
}

interface DeleteUserDisplayTextDefault extends Required<DeleteUserDisplayText> {}
```

Reply:
Just aligning with other components implementation, but this makes sense to me, happy to update across the board as followup.